cwiczenie3/osoba.php

```diff
@@ -5,22 +5,28 @@ class Osoba
     private $nazwisko;
     private $plec;
     private $rok;
-
     function __construct($imie, $nazwisko, $plec, $rok)
     {
         $this->imie = $imie;
         $this->nazwisko = $nazwisko;
         $this->plec = $plec;
         $this->rok = $rok;
     }
-
     public function powitaj()
     {
         return "Czesc mam na imie " . $this->imie;
     }
-
     public function obliczWiek()
     {
         return " i mam " . (date('Y') - ($this->rok)) . " lata";
     }
+
+    public function Imie()
+    {
+      return $this->imie;
+    }
+    public function dodajZnajomego(Osoba $osoba)
```

Comment:
Dobrze, fajnie! Zrob jeszcze jedna rzecz - zakladamy ze znajomych ma sie na cale zycie wiec to powinno byc zapisane wewnatrz klasy.